test/reftest.js

```diff
@@ -23,6 +23,7 @@ function reftestUnblock() {
 function doReftest() {
   if (reftestBlocked) return;
   if (doReftest.done) return;
+  if (skipTest.skipped) return;
```

Comment:
How about just using the existing `reportResultToServer.reported` here? Then it will also handle errors / exceptions too.

Reply:
(and you don't need skipTest.skipped at all)